full-stack-tests/imodelconnection-without-certa/src/Settings.ts

```diff
@@ -0,0 +1,149 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { LogLevel } from "@itwin/core-bentley";
+import { IModelReadRpcInterface } from "@itwin/core-common";
+import { TestUserCredentials } from "@itwin/oidc-signin-tool";
+import { ECSchemaRpcInterface } from "@itwin/ecschema-rpcinterface-common";
+import * as fs from "fs";
+import * as path from "path";
+
+export interface Backend {
+  version: string;
+  location: string;
+  name: string;
+  path: string;
+}
+
+export interface IModelData {
+  useName: boolean; // Defines whether or not to use the name of the iModel
+  id?: string; // The iModel Id - This is not required
+  name?: string; // The name is not required to actually get the iModel, only the id.
+  useITwinName: boolean;
+  iTwinId?: string;
+  iTwinName?: string;
+  changesetId?: string;
+}
+
+export function getRpcInterfaces() {
+  const rpcInterfaces = [];
+  rpcInterfaces.push(IModelReadRpcInterface);
+  rpcInterfaces.push(ECSchemaRpcInterface);
+
+  return rpcInterfaces;
+}
+
+export class Settings {
+  private _backend: Backend = {} as Backend;
+  public oidcClientId!: string;
+  public oidcScopes!: string;
+  public oidcRedirect!: string;
+  public oidcAuthority?: string;
+  public discovery!: string;
+  public gprid?: string;
+  public logLevel?: number;
+  public users: TestUserCredentials[] = [];
+
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  public get Backend(): Backend { return this._backend; }
+  public get user(): TestUserCredentials { return this.users[0]; }
+
+  public iModel: IModelData = {} as IModelData;
+
+  constructor() {
+    const dotenv = require("dotenv"); // eslint-disable-line @typescript-eslint/no-var-requires
+    const dotenvExpand = require("dotenv-expand"); // eslint-disable-line @typescript-eslint/no-var-requires
+
+    let envFile = path.join(__dirname, "..", "..", ".env")
+    if (!fs.existsSync(envFile)) {
+      console.log("Env file not found, trying other configuration.");
```

Comment:
What are we trying to fall back to here?  Either the .env should exist in the right place or not - I don't really want to look in more than one location...